apps/core/lib/core/snitch/address.ex

```diff
@@ -32,7 +32,7 @@ defmodule Core.Snitch.Address do
   @required_fields ~w(first_name last_name address_line_1 city zip_code)a
```

Comment:
I am kinda a skeptical with the variable being named `address_line_1`, IMO  any variable that relies on a numeral to distinguish it from a similar variable needs to be refactored. I think we should change the existing
name to something more specific.
Not exactly related to this but here is the article might give more insights:
Link: http://archive.oreilly.com/pub/post/the_worlds_two_worst_variable.html